ci/azure/posix.yml

```diff
@@ -73,24 +73,42 @@ jobs:
 
   steps:
     - script: |
-        if [ "$(uname)" == "Linux" ]; then
-          sudo apt-get update
-          sudo apt-get install -y libc6-dev-i386 $EXTRA_APT
-        fi
+        sudo apt-get update
+        sudo apt-get install -y libc6-dev-i386 $EXTRA_APT
       displayName: 'Install extra packages'
+      condition: eq('${{ parameters.name }}', 'Linux')
 
-    - script: echo '##vso[task.prependpath]$(HOME)/miniconda3/bin'
-      displayName: 'Set conda path'
+    - bash: echo '##vso[task.prependpath]$(HOME)/miniconda3/bin'
+      displayName: 'Add conda to PATH'
 
-    - script: ci/setup_env.sh
```

Comment:
so this is basically inlining setup_env?

Reply:
why are we doing this? that is called by every build

Reply:
We save about 2-3 min build time by using the CI's conda. And the Windows builds are not using the script. We probably could combine all Azure CI setups into one file. Now we have one for posix and one for windows. They are almost identical.

Reply:
ok the problem is there are *other* things that setup_env is doing which this is not. not averse to this, but we need to do it *inside* setup_env.sh